src/server/game/Achievements/AchievementMgr.cpp

```diff
@@ -2782,35 +2782,35 @@ void AchievementGlobalMgr::LoadRewards()
         AchievementEntry const* achievement = sAchievementStore.LookupEntry(entry);
         if (!achievement)
         {
-            sLog->outErrorDb("Table `achievement_reward` has wrong achievement (Entry: %u), ignore", entry);
+            sLog->outErrorDb("Table `achievement_reward` has wrong achievement (Entry: %u). Ignoring.", entry);
             continue;
         }
 
         AchievementReward reward;
-        reward.titleId[0]   = fields[1].GetUInt32();
-        reward.titleId[1]   = fields[2].GetUInt32();
+        reward.titleId[0]   = fields[1].GetUInt32(); // Alliance title
+        reward.titleId[1]   = fields[2].GetUInt32(); // Horde title
```

Comment:
I wanted to change these vars directly but they're used in the whole file, so too risky to change it all for me